core/client/fs/src/main/java/alluxio/client/file/FileSystem.java

```diff
@@ -141,16 +140,12 @@ public static FileSystem create(FileSystemContext context) {
           LOG.debug("{}={} ({})", key.getName(), value, source);
         }
       }
-      Class fsClass = conf.getClass(PropertyKey.USER_FILESYSTEM_CLASS);
```

Comment:
is it okay to get rid of this flag here? @calvinjia

Reply:
Looks good